catalog-rest-service/src/main/java/org/openmetadata/catalog/airflow/AirflowUtils.java

```diff
@@ -68,7 +69,7 @@ public static OpenMetadataIngestionComponent makeOpenMetadataDatasourceComponent
     Map<String, Object> dbConfig = new HashMap<>();
     dbConfig.put(INGESTION_HOST_PORT, databaseConnection.getHostPort());
     dbConfig.put(INGESTION_USERNAME, databaseConnection.getUsername());
-    dbConfig.put(INGESTION_PASSWORD, databaseConnection.getPassword());
+    dbConfig.put(INGESTION_PASSWORD, decryptIfTokenized(databaseConnection.getPassword()));
```

Comment:
can we do this only when the role is admin or bot?

Reply:
done.